### ISSUE TITLE
Modal opens too slowly

### ISSUE THREAD

**Issue by klendi** [AstrOOnauta/react-native-international-phone-number#52]
Is there any way to make it without animations or faster ?

**Comment by AstrOOnauta** [comment]
@klendi sorry about this...but I dont know what is happining there...

I use this lib in large and complex projects (apps with +130MB and +50k users daily) and it works fine here.

So, check rerenders in your application or check if this happens with another library. Let me know if this trouble continues..

🚀 AstrOOnauta 🚀

**Comment by klendi** [comment]
I mean it's not terribly slow, just opens with a delay of 500ms or something instead of instant. It doesn't happen with another library I have tried with `react-native-country-picker-modal` and it's instant and super fast. Maybe its the animation ?

**Comment by AstrOOnauta** [comment]
@klendi yes, it has an animation when opening the modal...

I undestand, i thought it was something very serious, i understand...thank you...

**Comment by klendi** [comment]
Yeah but that 1s delay is too slow anyway to disable it or configure it would be great

**Comment by fbele** [comment]
I also think that the modal should be a bit more responsive, faster.
But the module overall is a great work!

**Comment by flodlc** [comment]
Same here, modal should open without delay. Great library !

**Comment by tiagoapp** [comment]
Is it possible to get ride of this delay somehow? @AstrOOnauta

**Comment by AstrOOnauta** [comment]
I release a new version fixing this issue. The old country select didnt allow to change the delay to open the modal, so I updated to other country select component.

So, please, update your lib for the latest version v.0.10.0 and let me know if you have more troubles.

🚀 AstrOOnauta 🚀
